install/kubernetes/cilium/values.yaml

```diff
@@ -117,8 +117,9 @@ tolerations:
   #   value: "value"
   #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
 
-# -- Node labels for agent pod assignment
+# -- Node labels for agent pod assignment. Deprecated, please use the option for respective component (e.g. operator.nodeSelector).
 # ref: https://kubernetes.io/docs/user-guide/node-selection/
+## TODO: Remove this option in 1.10 to avoid breaking change
```

Comment:
I assume you mean remove this option in 1.11?

Reply:
oh I planned to backport and removed this one in next PR :)